src/app.ts

```diff
@@ -0,0 +1,373 @@
+import './index.css';
+import MainPage from './pages/main/main';
+import { CartPage, getCurrPage, getPerPage, perPage, setCurrPage, setPerPage, updatePaginParam } from './pages/cart/cart';
+import Page from './pages/templates/page';
+import DescriptionPage from './pages/description/description';
+import ErrorPage from './pages/error404/error404';
+import Header from './pages/components/header/header';
+import Footer from './pages/components/footer/footer';
+import { ItemCart } from './pages/components/itemCart/itemCart';
+import { IPrototypeItem } from './pages/templates/items';
+import shoes from './db/shoes';
+import iconsSVG from './pages/templates/icons';
+import { Form } from './pages/components/form/form';
+import { removeSearchParams } from './pages/templates/filters';
+
+//let itemsAddCartButton: NodeList;
```

Comment:
All comments like this shouldn't be there in the first place, I guess it's purpose only for debugging, you shouldn't store them